rpcs3/rpcs3qt/update_manager.cpp

```diff
@@ -96,7 +107,18 @@ void update_manager::check_for_updates(bool automatic, bool check_only, bool aut
 		Q_EMIT signal_update_available(result_json && !m_update_message.isEmpty());
 	});
 
+#ifdef __APPLE__
+	const std::string url = "https://update.rpcs3.net/?api=v3&c=" + rpcs3::get_commit_and_hash().second +
+	"&os_type=macos" +
+#ifdef ARCH_X64
+	"&os_arch=x64" +
+#elifdef ARCH_ARM64
+	"&os_arch=arm64" +
+#endif
+	"&os_version=" + fmt::format("%i.%i.%i", Darwin_Version::getNSmajorVersion(), Darwin_Version::getNSminorVersion(), Darwin_Version::getNSpatchVersion());
+#else
 	const std::string url = "https://update.rpcs3.net/?api=v2&c=" + rpcs3::get_commit_and_hash().second;
```

Comment:
does the v3 api not support empty query params or why is this not using v3 as well?